### PR TITLE
Feat: Add Search and Browse Screen

### PR DESCRIPTION
**Pull request by dikshamodi**

### CODE REVIEW

File: frontend/src/Pages/Customer/ViewShops.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
+import axios from 'axios';
+import React from 'react';
+import { Col, Button, Card, Container, Form, ListGroup } from 'react-bootstrap';
+import { Link } from "react-router-dom";
+import '../../App.css';
+import LocationInput from '../LocationInput';
+
+const SEARCH_QUERY_URL = "https://speedy-anthem-217710.an.r.appspot.com/api/query/elastic";
+const BROWSE_QUERY_URL = "https://speedy-anthem-217710.an.r.appspot.com/api/browse/elastic";
```
Comment (onish-garg) [inline frontend/src/Pages/Customer/ViewShops.js:9]: Consider putting these in `/src/routes.js`

Comment (dikshamodi) [reply frontend/src/Pages/Customer/ViewShops.js]: Done. Thank you!!! Left the viewCatalog one for now. Will change it when I add buyercatalog

File: frontend/src/Pages/Customer/ViewShops.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+              <Col xs={7}>
+                <Form.Control
+                  placeholder="Search Nearby"
+                  autoComplete="off"
+                  onChange={e => {
+                    this.setState({ searchQuery: e.target.value }, () => {
+                      if (this.state.searchQuery === "") {
+                        this.updateBrowseResults();
+                      }
+                    })
```
Comment (harshil93) [inline frontend/src/Pages/Customer/ViewShops.js:114]: Better to create a function for multi lines lambdas

Comment (dikshamodi) [reply frontend/src/Pages/Customer/ViewShops.js]: Done. Thank you!!

**Pull request merged**